test/basic.js

```diff
@@ -53,7 +53,7 @@ t.test('spawn execPath', function (t) {
 })
 
 t.test('exec shebang', function (t) {
-  var child = cp.exec(fixture + ' xyz')
+  var child = spawn(fixture, ['xyz'])
```

Comment:
Yeah, apparently signal-exit just straight up does not work on Linux.  This is strange.